app/Http/Resources/TestQuestion.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class TestQuestion extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'score' => $this->pivot->score,
+            'topic' =>  Topic::make($this->topic),
```

Comment:
You should create a TopicShort resource with id and name only and use it instead of the detailed resource.